### PR TITLE
bump automation for e2e-testing

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Add e2e-testing

## Why is it important?

Then https://github.com/elastic/e2e-testing/pull/2198 won't be needed

### CODE REVIEW

File: .ci/.bump-stack-release-version.yml [hunk 1]
```diff
@@ -49,3 +49,11 @@ projects:
     enabled: true
     labels: dependency
     title: "chore: update elastic stack release version"
+  - repo: e2e-testing
+    script: .ci/bump-stack-release-version.sh
+    branches:
+      - main
```
Comment (mdelapenya) [inline .ci/.bump-stack-release-version.yml:55]: Do we need main here? I expect to update the stale version for the last minor, only

Comment (v1v) [inline .ci/.bump-stack-release-version.yml:55]: This implementation does not use aliases, so we cannot use `8.<minor>` yet, so I'll hardcode this for the time being, then we can implement this feature, so I guess the last minor is `8.0` now?

```suggestion
      - "8.0"
```

Comment (v1v) [reply .ci/.bump-stack-release-version.yml:55]: @mdelapenya , what do you think?

Comment (mdelapenya) [reply .ci/.bump-stack-release-version.yml:55]: These are the branches we currently support: main, 8.1, 8.0 and 7.17.

OTOH, there is no `.ci/bump-stack-release-version.sh` file in the repo, is this something we need to add?

Comment (v1v) [reply .ci/.bump-stack-release-version.yml:55]: Yes, we need to do so, I guess the changes in https://github.com/elastic/e2e-testing/pull/2198 could be moved to the script `.ci/bump-stack-release-version.sh`

what do you think?

**Review by mdelapenya — APPROVED** [review]
We only need to check what versions need to run this automation. In the meantime LGTM

File: .ci/.bump-stack-release-version.yml [hunk 2]
```diff
@@ -49,3 +49,13 @@ projects:
     enabled: true
     labels: dependency
     title: "chore: update elastic stack release version"
+  - repo: e2e-testing
+    script: .ci/bump-stack-release-version.sh
+    branches:
+      - "8.1"
```
Comment (v1v) [inline .ci/.bump-stack-release-version.yml:55]:
```suggestion
      - "main"
      - "8.1"
```

File: .ci/.bump-stack-release-version.yml [hunk 3]
```diff
@@ -49,3 +49,14 @@ projects:
...
     labels: dependency
     title: "chore: update elastic stack release version"
+  - repo: e2e-testing
+    script: .ci/bump-stack-release-version.sh
+    branches:
+      - "main"
+      - "8.1"
+      - "8.0"
+      - "7.17"
+      - "7.17"
```
Comment (v1v) [inline .ci/.bump-stack-release-version.yml:59]:
```suggestion
      - "7.17"
```

Comment (v1v) [inline .ci/.bump-stack-release-version.yml:59]:
```suggestion
```

**Review by cachedout — APPROVED** [review]

**Comment by v1v** [comment]
As we are moving away from jenkins specific, I'll close this PR now

**Pull request closed without merging**